idaes/models/properties/modular_properties/state_definitions/FpcTP.py

```diff
@@ -142,7 +142,7 @@ def mole_frac_comp(b, j):
 
     b.mole_frac_phase_comp = Var(
         b.phase_component_set,
-        bounds=(1e-20, 1.001),
+        bounds=(1e-50, 1.001),
```

Comment:
I am not sure about this change, at least at in the general code. Firstly, for most applications 1e-50 is indistinguishable from 1e-20 (which I suppose is an argument that the change doesn't matter). However, it does suggest that we really should allow users to set the bounds on mole fractions in the configuration which I think we currently do not.

Secondly, if we change the value here that would suggest we should be changing it in all the other state definitions too for consistency there.

Reply:
In [idaes/models/properties/modular_properties/base/generic_property.py](https://github.com/IDAES/idaes-pse/blob/main/idaes/models/properties/modular_properties/base/generic_property.py) line 4688-4696, the bounds of `log_mole_frac_phase_comp` was set to be `(-100, log(1.001))`. This is a declaration before `mole_frac_phase_comp` in `FpcTP`. The inconsistency in bounds caused a warning that the value of `mole_frac_phase_comp` is out of bounds when we do the initialization of low concentration solubility problem (some inlet component fraction was set at 1e-20). 

Since e^(-100) is approximately 4e-44, I put 1e-50 there.

And yes, I think all bounds in state definitions should be changed consistently, or we can remove one between the original var and the log-transformed var.